invadedhouse/run.py

```diff
@@ -21,9 +20,9 @@
 def main():
     cprint('   ____                       __         __  __ __                   ', 'red')
     cprint('  /  _/___   _  __ ___  _ ___/ /___  ___/ / / // /___   __ __ ___  ___', 'yellow')
-    cprint(' _/ / / _ \ | |/ // _  `// _  // -_)/ _  / / _  // _ \ / // /(_-</  -_)', 'green')
-    cprint('/___//_//_/ |___/ \_,_/ \__,_/ __/ \_,_/  /_//_/ ___/ \__,_//___/ \__/ ', 'cyan')
-    print('')
+    cprint(' _/ / / _ \\ | |/ // _  `// _  // -_)/ _  / / _  // _ \\ / // /(_-</  -_)', 'green')
+    cprint('/___//_//_/ |___/ \\_,_/ \\__,_/ __/ \\_,_/  /_//_/ ___/ \\__,_//___/ \\__/ ', 'cyan')
+    print()
```

Comment:
HMMMMMMMMMMMMMMMMMMMMMMMMMM? WHAT? WHAT? WHAT THIS MEAN IT LOOK WIERD AND WHY DID YOU ADD IN THE DOUBLE SLAHSES AGAIN? ehh okay!

Reply:
the letter `\` is called an escape character... (remember how we use it for `\'` when we want to use quote characters? So to use it properly in a string you have to do what's called "double escape"... it'll look fine on screen when the program runs